Add `ReduceMotionConfig` docs (#6165)
software-mansion/react-native-reanimated: packages/docs-reanimated/docs/components/ReducedMotionConfig.mdx

---
id: ReducedMotionConfig
title: ReducedMotionConfig
sidebar_label: ReducedMotionConfig
sidebar_position: 1
---

> The `ReducedMotionConfig` component determines the default animation behavior in response to the device's reduced motion accessibility setting. It affects your application globally. The default behavior disables all animation if reduced motion is enabled on a device. You can utilize this component to override that behavior. If you wish to fine-tune it for your specific case, learn more about [Accessibility](../guides/accessibility) and [`useReducedMotion`](../device/useReducedMotion) in Reanimated.

Reviewer: ```suggestion
`ReducedMotionConfig` component let's you change behavior in response to the device's reduced motion accessibility setting. By default it disables all animation when the reduced motion is enabled on a device. You can adjust it for your specific use case. You can learn more about [Accessibility](/docs/guides/accessibility) and [`useReducedMotion`](/docs/device/useReducedMotion) in Reanimated.
```
